### ISSUE TITLE
unable to connect to extension marketplace

### ISSUE THREAD

**Issue by luboi75** [VSCodium/vscodium#764]
after recent update (12th July 2021) I get "We cannot connect to the Extension Marketplace at this time, please try again later."

**to reproduce:**
open vscode, 
switch to extensions, 
type any string in the search box

**expected behavior:**
connection to marketplace works fine, search shows results

**version information**
Version: 1.58.0
Commit: 2d23c42a936db1c7b3b06f918cde29561cc47cd6
Date: 2021-07-12T23:51:36.303Z
Electron: 12.0.14
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Linux x64 5.10.49-1-MANJARO

**additional info - from developer console**
Access to XMLHttpRequest at 'https://open-vsx.org/vscode/gallery/extensionquery' from origin 'vscode-file://vscode-app' has been blocked by CORS policy: Request header field x-market-user-id is not allowed by Access-Control-Allow-Headers in preflight response.
extensionsViews.ts:323 Error querying extensions gallery XHR failed
query @ extensionsViews.ts:323
async function (async)
query @ extensionsViews.ts:320
(anonymous) @ extensionsViews.ts:241
y @ async.ts:24
show @ extensionsViews.ts:239
(anonymous) @ extensionsViewlet.ts:693
onDidAddViewDescriptors @ extensionsViewlet.ts:692
(anonymous) @ viewPaneContainer.ts:531
fire @ event.ts:583
triggerOnDidAddVisibleViewDescriptors @ viewContainerModel.ts:447
onDidChangeContext @ viewContainerModel.ts:634
(anonymous) @ viewContainerModel.ts:329
fire @ event.ts:583
(anonymous) @ event.ts:73
fire @ event.ts:583
fire @ event.ts:713
setContext @ contextKeyService.ts:326
set @ contextKeyService.ts:200
doSearch @ extensionsViewlet.ts:680
(anonymous) @ extensionsViewlet.ts:645
(anonymous) @ async.ts:261
Promise.then (async)
trigger @ async.ts:255
triggerSearch @ extensionsViewlet.ts:645
(anonymous) @ extensionsViewlet.ts:552
fire @ event.ts:585
(anonymous) @ suggestEnabledInput.ts:173
(anonymous) @ textModel.ts:252
fire @ event.ts:583
endDeferredEmit @ textModel.ts:3567
pushEditOperations @ textModel.ts:1271
_innerExecuteCommands @ cursor.ts:787
executeCommands @ cursor.ts:745
_executeEditOperation @ cursor.ts:448
(anonymous) @ cursor.ts:655
_executeEdit @ cursor.ts:608
type @ cursor.ts:644
(anonymous) @ viewModelImpl.ts:983
_withViewEventsCollector @ viewModelImpl.ts:1050
_executeCursorEdit @ viewModelImpl.ts:969
type @ viewModelImpl.ts:983
_type @ codeEditorWidget.ts:1081
type @ codeEditorWidget.ts:1607
type @ viewController.ts:70
(anonymous) @ textAreaHandler.ts:266
fire @ event.ts:583
(anonymous) @ textAreaInput.ts:356
request.ts:42 POST https://open-vsx.org/vscode/gallery/extensionquery net::ERR_FAILED

**Comment by daiyam** [comment]
Hi, it's a duplicate of #746.

**Comment by luboi75** [comment]
thanx, sorry for duplicating (age confused me)
